OWNERS_ALIASES

```diff
@@ -4,9 +4,8 @@ aliases:
   # active folks who can be contacted to perform admin-related
   # tasks on the repo, or otherwise approve any PRS.
   controller-runtime-admins:
-  - droot
-  - mengqiy
-  - pwittrock
```

Comment:
should we not add those to the emeritus ? (example: https://github.com/kubernetes-sigs/kubebuilder/blob/master/OWNERS_ALIASES#L22-L29)

Reply:
Good point, done!